Cargo.toml

```diff
@@ -4,6 +4,8 @@ version = "0.1.0"
 edition = "2021"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[package.metadata.configure_me]
```

Comment:
A few notes on the commit message here: 
* I wouldn't really classify this as a refactor, as we're adding a new dependency + changing the way we handle args (can remove that prefix)
* "no longer needed. needed." -> repeat/ typo in there.

Reply:
Thank yiu